README.md

```diff
@@ -62,175 +66,48 @@ The MEI takes a different approach. With the customization system, schemas may b
-
-   * If you do not have a clone on your local machine yet run the following from your command line:
-
-     ```shell
-     git clone https://github.com/music-encoding/music-encoding.git --recursive
-     ```
-
-   * If you already have a clone on your system you still might have to initialize the submodules by running the following commands from the command line:
-
-     Switch to your clone's directory:
-
-     ```shell
-     cd [YOUR-CLONE-LOCATION]
-     ```
-
-     Initialize the submodules:
-
-     ```shell
-     git submodule init
-     ```
-
-     Update the submodules:
-
-     ```shell
-     git submodule update
-     ```
-
-2. Check if your system meets the build requirements
-
-   * Is Java 8 or above available on your machine?
-
-     Java 8 or above is needed for the build process driven by apache Ant (see below).
-     To check for Java on your machine, run the following command:
-
-     ```shell
-     java -version
-     ```
-
-     This should return something similar to:
-
-     ```shell
-     openjdk version "11.0.9" 2020-10-20
-     OpenJDK Runtime Environment (build 11.0.9+11)
-     OpenJDK 64-Bit Server VM (build 11.0.9+11, mixed mode)
-     ```
-
-     If the version number indicated is lower than `8.0.0` or if the command returns an empty string, please update or install Java according to an installation instruction matching your operating system (to be found on the internet).
-
-   * Is Apache Ant installed?
-
-     [Apache Ant](https://ant.apache.org/manual/install.html) is a library for building software projects and drives the creation of MEI schemata and guidelines from the ODD source files.
-
-     Run the following command to see if it is available on your system:
-
-     ```shell
-     ant -version
-     ```
-
-     This should return something similar to:
-
-     ```shell
-     Apache Ant(TM) version 1.10.9 compiled on September 27 2020
-     ```
-
-     We recommend using version 1.10.9, being the latest stable release of Apache Ant. If your system has an older version of Apache Ant installed you might still give it a try though. If the prompt returns an empty string, please refer to the [Apache Ant Installation Instructions](https://ant.apache.org/manual/install.html).
-
-3. Initialize the build process
-
-   * Switch to your clone’s directory:
-
-     ```shell
-     cd [YOUR-CLONE-LOCATION]
-     ```
+Generally, the build process for MEI artifacts is driven by [Apache Ant™](https://ant.apache.org), a Java library and command line tool. Depending on what artifacts you want to build the prerequisites against your operating will increase.
```

Comment:
Something is missing in: 

`the prerequisites against your operating will increase`

Reply:
`Generally` is repeated in line 78